core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -1037,7 +1037,8 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
     addAndVerifyAcls(Set(new AccessControlEntry(clientPrincipalString, WildcardHost, READ, ALLOW)), topicResource)
 
     // in this case, we do an explicit seek, so there should be no need to query the coordinator at all
-    val consumer = createConsumer()
+    // remove the group.id config to avoid coordinator created
+    val consumer = createConsumer(configsToRemove = List(ConsumerConfig.GROUP_ID_CONFIG))
```

Comment:
Same as above.



This test case is to test manual assignment without storing offsets. So we should not create groupID (the third parameter from the end)

test added for this issue: https://issues.apache.org/jira/browse/KAFKA-4034